cmake/sdp.cmake

```diff
@@ -4,6 +4,24 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+function(sdp_assembly hrt_srcs)
+  sdp_assembly_generate("${CMAKE_SOURCE_DIR}/${hrt_srcs}")
```

Comment:
please allow for `hrt_srcs` to have absolute path.

Do not assume that that path is always relative.

Reply:
seems my comment was misread.

I asked for the possibility for `hrt_srcs` to have absolute path, not require it to have absolute path.
CMake has the test: `if(IS_ABSOLUTE <path>)` https://cmake.org/cmake/help/latest/command/if.html#is-absolute which can be used to test if path should be converted when it's relative.

Note, requiring an absolute path always is also acceptable, so this comment in non-blocking.

Reply:
Fully acceptable, only reason for my followup comment was to clarify the intent of my original comment.